website/docs/configuration/ort-yml.md

```diff
@@ -195,6 +195,10 @@ resolutions:
   - message: "Does not have X.*"
     reason: "BUILD_TOOL_ISSUE"
     comment: "Error caused by a known issue for which a fix is being implemented, see https://github.com/..."
+  
+  - message: "NOTICE_DEFAULT"
+    reason: "BUILD_TOOL_ISSUE"
+    comment: "This is expected when no third-party license data is collected and a default notice file is still generated."
```

Comment:
The entries here refer to so call `OrtIssue`s. 
These are entries contained in an `OrtResult` file. 
The output of the reporter are not `OrtIssue`s, so adding this entry does not make sense.